Week-4/q_2.js

```diff
@@ -0,0 +1,12 @@
+class Person {
```

Comment:
Where is the derived class or inherited class ? read the requirements again and chaine the implementation